video2dataset/worker.py

```diff
@@ -36,6 +36,8 @@ def __init__(
         oom_shard_count,
         encode_format,
         retries,
```

Comment:
we don't use this param in worker at all

Reply:
yeah can remove for now